src/server/ua_services_monitoreditem.c

```diff
@@ -215,8 +215,11 @@ checkAdjustMonitoredItemParams(UA_Server *server, UA_Session *session,
                 params->samplingInterval = vn->minimumSamplingInterval;
             UA_NODESTORE_RELEASE(server, node);
         }
+        else{
+          params->samplingInterval = server->config.samplingIntervalLimits.min;
```

Comment:
@jpfr This is maybe not correct! Which plugin should be requested in this case? NODESTORE_GET_EX?